src/main/java/com/example/bytex/user/UserEntityController.java

```diff
@@ -0,0 +1,60 @@
+package com.example.bytex.user;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.List;
+
+@RestController
+@RequestMapping(path = "users")
+public class UserEntityController {
+
+    UserEntityService userEntityService;
+
+    @Autowired
+    public UserEntityController(UserEntityService userEntityService) {
+        this.userEntityService = userEntityService;
+    }
+
+    @GetMapping
+    public ResponseEntity<List<UserEntity>> getUsers(){
+        List<UserEntity> allUsers = userEntityService.getUsers();
+
+        if(allUsers.size() != 0)
+        {
+            return new ResponseEntity<>(allUsers, HttpStatus.OK);
+        }
+
+        throw new ResponseStatusException(HttpStatus.NO_CONTENT, "Nothing to show");
```

Comment:
While this is a valid approach, most apis would just return an empty list in this scenario, not an exception

Reply:
done